tests/saliency_metrics/metrics/insertion_deletion/test_insertion_deletion_metric.py

```diff
@@ -0,0 +1,110 @@
+import numpy as np
+import pytest
+import torch
+import torch.nn as nn
+
+from saliency_metrics.metrics.insertion_deletion.insertion_deletion_metric import InsertionDeletion
+from saliency_metrics.models import CUSTOM_CLASSIFIERS
+
+
+@CUSTOM_CLASSIFIERS.register_module()
+class TestNet(nn.Module):
+    def __init__(self) -> None:
+        super().__init__()
+        self.features = nn.Sequential(
+            nn.Conv2d(in_channels=3, out_channels=5, kernel_size=3, padding=1),
+            nn.MaxPool2d(2, 2),
+            nn.ReLU(inplace=True),
+        )
+        self.classifier = nn.Sequential(nn.Linear(16 * 16 * 5, 10))
+        self.initialize_weights()
+
+    def forward(self, x: torch.Tensor) -> torch.Tensor:
+        x = self.features(x)
+        x = x.view(-1, 16 * 16 * 5)
+        x = self.classifier(x)
+        return x
+
+    def initialize_weights(self) -> None:
+        for i, x in enumerate(self.parameters()):
+            prng = np.random.RandomState(i)
```

Comment:
As I said, the numbers are not constants if you draw them from a Uni distribution. Secondly, the standard way to initialize weights in PyTorch is to use `nn.init` and `Module.apply` rather than sample a random `ndarray`, convert it to `tensor` and assign it.

What you can do here is directly intialize the weight of conv and fc layer to **constants** like 1.0 or 2.0 and set the bias to false.